draft-gpew-priv-ppm.md

```diff
@@ -854,29 +866,28 @@ input_share = context.Open(ReportShare.nonce || ReportShare.extensions,
 ~~~
 
 where `sk` is the HPKE secret key, `task_id` is the task ID, and `nonce` and
-`extensions` are the nonce and extensions of the report share respectively. If
-decryption fails, then the aggregator fails with error `hpke-decrypt-error`.
-Variable `server_role` is the Role of the intended recipient.
+`extensions` are the nonce and extensions of the report share respectively.
+Variable `server_role` is the Role of the intended recipient. If decryption
+fails, then the aggregator MUST fail with error `hpke-decrypt-error`.
 
 Next, the aggregator runs the preparation-state initialization algorithm for the
 VDAF associated with the task and computes the first state transition. Let
 `agg_param` denote the aggregation parameter, and let `input_share` denote the
 input share:
 
 ~~~
-prep_state = PrepState(vdaf_verify_param,
-                       agg_param, nonce, input_share)
-out = prep_state.next(None)
+state = prep_init(vdaf_verify_param, agg_param, nonce, input_share)
+out = prep_next(state, None)
 ~~~
-[CP: Note that this syntax is based on
-https://github.com/cjpatton/vdaf/pull/14.]
 
-If either step fails, then the aggregator fails with error `vdaf-prep-error`.
-Otherwise, `out` is interpreted as follows. If the VDAF is 0-round, then `out`
-is the aggregator's output share, in which case the aggregator finishes and
-stores its output share for further processing as described in
-{{out-to-agg-share}}. Otherwise, if the VDAF consists of one round or more, then
-the aggregator continues and `out` is the aggregator's next VDAF message.
+If either step fails, then the aggregator MUST fail with error
+`vdaf-prep-error`. Otherwise, `out` is interpreted as follows. If the VDAF is
+0-round, then `out` is the aggregator's output share, in which case the
+aggregator finishes and stores its output share for further processing as
+described in {{out-to-agg-share}}. Otherwise, if the VDAF consists of one round
```

Comment:
This splitting of VDAF output into two parts is new to me -- how is this to be done? e.g. is the definition of how to split the output into a tuple part of the VDAF?

Reply:
Yeah, this is just a matter of aligning the notation with the current draft: https://cjpatton.github.io/vdaf/draft-patton-cfrg-vdaf.html#name-preparation